releasenotes/notes/support-sprase-pauli-op-00d29000d48c93ca.yaml

```diff
@@ -0,0 +1,13 @@
+---
+features:
+  - |
+    Updated :func:`~.to_ising` to support ``qiskit.quantum_info.SparsePauliOp``.
+    Added ``opflow`` argument to :func:`~.to_ising` to control the output is
+    opflow operator (if ``True``) or ``SparsePauliOp`` (if ``False``).
+    The default value of ``opflow`` argument is currently ``True``, but it will
+    first be changed to ``False`` and then deprecated in future releases.
+  - |
+    Updated :func:`~.from_ising` to support ``qiskit.quantum_info.SparsePauliOp``.
+    ``qubit_op`` argument can currently accept opflow operators,
+    but it has been superseded by ``SparsePauliOp``.
+    Opflow will be deprecated in a future release and subsequently removed after that.
```

Comment:
This text is similar again to the suggestion made for the from_ising note. So again is thats ok probably we want the same/similar here too.

Reply:
Thanks. I include it. cc41551